docs/argo-server-sso.md

```diff
@@ -188,6 +188,25 @@ sso:
   userInfoPath: /oauth2/v1/userinfo
 ```
 
+## Filter groups
+
+> v3.5 and after
+
+```yaml
```

Comment:
the YAML block should be after the description of the field, to be consistent with the rest of the page